charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -212,8 +203,6 @@ spec:
             value: "no"
           {{- end }}
        {{- if .Values.medusa.enabled }}
-          - name: JVM_EXTRA_OPTS
```

Comment:
When removing this line, the above "Env" part must be fixed also (it's no longer enough to have either medusa or reaper enabled to render the env)